### DISCUSSION TITLE
NULL VALUE

### DISCUSSION THREAD

**Discussion by George-Zambrano96** [dbeaver/dbeaver#13578] [General]
Hey I can't for the life of me figure out why the new column labeled as "TOTALDEATHPERCENTAGE" is coming out as Null. Any help would be appreciated thank you!
![PM Screen Shot 2021-08-18 at 9 52 28](https://user-images.githubusercontent.com/88901339/129995239-30601754-3e95-4d2c-8f14-a2b6279ed136.png)

**Comment by p91paul** [comment]
from your screenshot I can see that total_cases and total_deaths are varchar columns. You should probably convert them to number in your totaldeathspercentage formula!

**Reply by George-Zambrano96** [reply]
You are correct, it seems that all of the columns were imported as strings. Do i have to change each individually using either convert or cast functions or is there a way to do it all at once? Thanks for the reply.

**Reply by p91paul** [reply]
If you plan to reuse that table, I suggest you drop it, recreate it manually (i.e. not using the import functionality) with correct datatypes, and then once it's created reimport data in it. There might be some functionality to set datatypes when you import data to a new table, but I don't usually do data imports so I can't help.
I also doubt you can change datatype for an existing column with data inside it.
